transformers/component_tests/queries/get_urn_query_test.go

```diff
@@ -1,157 +0,0 @@
-package queries
-
-import (
-	"math/rand"
-	"strconv"
-
-	storage_helper "github.com/makerdao/vdb-mcd-transformers/transformers/storage/test_helpers"
-
-	"github.com/makerdao/vdb-mcd-transformers/test_config"
-	"github.com/makerdao/vdb-mcd-transformers/transformers/component_tests/queries/test_helpers"
-	"github.com/makerdao/vdb-mcd-transformers/transformers/storage/vat"
-	"github.com/makerdao/vdb-mcd-transformers/transformers/test_data"
-	"github.com/makerdao/vulcanizedb/pkg/core"
-	"github.com/makerdao/vulcanizedb/pkg/datastore/postgres/repositories"
-	. "github.com/onsi/ginkgo"
-	. "github.com/onsi/gomega"
-)
-
-var _ = Describe("Single urn view", func() {
-	var (
-		vatRepo                vat.VatStorageRepository
-		headerRepo             repositories.HeaderRepository
-		urnOne, urnTwo         string
-		blockOne, timestampOne int
-		headerOne              core.Header
-		diffID                 int64
-	)
-
-	const getUrnQuery = `SELECT urn_identifier, ilk_identifier, ink, art, created, updated FROM api.get_urn($1, $2, $3)`
```

Comment:
I like the idea of using the `api.urn_snapshots` table to replace this query, but do you know how we'd structure the query against that table to guarantee we're getting the most recent one by block number in GraphQL?

Reply:
Worst come to worst I could see a case for keeping the query and it just wraps the `order by block_number desc limit 1`

Reply:
I guess it's a question if people are using it - we could certainly use the query you just said to keep it around.

Reply:
I restored get_urn - assuming all_urns only returns the latest unique urns (ergo distinct on urn id and ilk) then get_urn should always return only 1 urn. 

I'd like to have the equivalent of ASSERT somewhere here - since if it returns more than one then something is wrong in the all_urns query. In either case as long as get_urn uses all_urns then we shouldn't need to `order by block_number desc limit`.

Reply:
(Assuming I got it right of course).